templates/terraform/constants/health_check.erb

```diff
@@ -0,0 +1,77 @@
+// Whether the port should be set or not
+func validatePortSpec(diff *schema.ResourceDiff, blockName string, defaultPort int) error {
+    block := diff.Get(blockName+".0").(map[string]interface{})
+    portSpec := block["port_specification"]
+    portName := block["port_name"]
+    port := block["port"]
+
+    incorrectPort := (port != nil && port != 0)
```

Comment:
nit - maybe "hasPort"? I'm not sure I understood what incorrectPort meant immediately